tasks/install_agent.json

```diff
@@ -0,0 +1,55 @@
+{
+  "description": "Install the OpenVox agent package",
+  "parameters": {
+    "version": {
+      "description": "The version of openvox-agent to install (defaults to latest when no agent is installed)",
+      "type": "Optional[String[1]]"
+    },
+    "collection": {
+      "description": "The OpenVox collection to install from (defaults to openvox8, which maps to the latest collection released)",
+      "type": "Optional[Enum[openvox7, openvox8]]"
```

Comment:
Does the enum need to contain `nightly`? And I think the logic in the task itself is easier if the default is already set in the json?

Reply:
At the time, OpenVox hasn't any nightly builds. But default to json it's ok.

Reply:
I had a look at the code. And neither version, collection or nightly can get a default.
- nightly: is overridden if nightly is included in collection parameter. The enum doesn't have this collections at the time because OpenVox doesn't offer nightly builds, yet. I will remove this nightly stuff since we have nightly builds.
- collection: if the puppet agent is installed and NO collection is set, the major version of the puppet agent is taken
- version: default to latest would result in an update every runtime

Reply:
done